packages/react-components/react-toolbar/src/components/ToolbarGroup/useToolbarGroupStyles.styles.ts

```diff
@@ -6,19 +6,11 @@ export const toolbarGroupClassNames: SlotClassNames<ToolbarGroupSlots> = {
   root: 'fui-ToolbarGroup',
 };
 
-/**
- * Styles for the root slot
- */
-const useStyles = makeStyles({
```

Comment:
you need to removethe import of `makeStyles` to make the linter happy

Reply:
@benlemmon-ms - FYI I removed the import in this PR to fix the build error.